src/render/renderer.zig

```diff
@@ -31,13 +31,21 @@ pub const Renderer = extern struct {
     pub const clear = wlr_renderer_clear;
 
     extern fn wlr_renderer_init_wl_display(r: *Renderer, server: *wl.Server) bool;
-    pub const initServer = wlr_renderer_init_wl_display;
+    pub fn initServer(r: *Renderer, server: *wl.Server) !void {
+        if (!wlr_renderer_init_wl_display(r, server)) {
+            return error.Failure;
```

Comment:
I think we should be a little less generic with the error messages.
```suggestion
            return error.RendererInitFailure;
```